tasks/redirects.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function( grunt ) {
+
+var wp = require( "wordpress" );
+
+grunt.registerTask( "deploy-redirects", function() {
+	var config = grunt.config( "wordpress" );
+	var redirects = grunt.file.exists( "redirects.json" ) ? grunt.file.readJSON( "redirects.json" ) : {};
+	var client = wp.createClient( config );
+
+	client.authenticatedCall( "jq.setRedirects", JSON.stringify( redirects ), this.async() );
```

Comment:
Or is this just because we can catch syntax errors in the file more easily this way?